### DISCUSSION TITLE
ClusterGVis包对接单细胞遇到的bug

### DISCUSSION THREAD

**Discussion by zhanghaodent** [junjunlab/ClusterGVis#34] [General]
在使用prepareDataFromscRNA函数完成对单细胞数据处理，然后进行visCluster时时遇到了图片中以下报错，想请教老师和各位大神
Error in ComplexHeatmap::anno_block(align_to = align_to, panel_fun = function(index,  : 
  unused arguments (align_to = align_to, panel_fun = function(index, nm) {
    npos = as.numeric(unlist(strsplit(nm, split = "C"))[2])
    grid::grid.rect(gp = grid::gpar(fill = colanno[npos], col = NA))
    if (annnoblock.text == TRUE) {
        grid::grid.text(label = paste("n:", length(index), sep = ""), rot = 90, gp = grid::gpar(col = annnoblock.gp[1], fontsize = as.numeric(annnoblock.gp[2])))
    }
})

**Comment by xgz432501** [comment]
遇到了同样的报错，有大神解决了吗

**Reply by junjunlab** [reply]
安装最新的complexheatmap

**Reply by xgz432501** [reply]
谢谢，跑成功了

**Comment by Zhaoyangyang-NCU** [comment]
运行enrich <- enrichCluster(object = scRNA,
                        OrgDb = org.Mm.eg.db,
                        type = "BP",
                        organism = "mmu",
                        pvalueCutoff = 0.5,
                        topn = 5,
                        seed = 5201314)
报错！

Error: Cannot find 'wide.res' in this Seurat object
